learn/glossary.md

```diff
@@ -36,3 +45,13 @@ This page for instance, not being a normative document, is not able to proscribe
 #### See also
 
 * [normative](https://developer.mozilla.org/en-US/docs/Glossary/Normative) and [non-normative](https://developer.mozilla.org/en-US/docs/Glossary/non-normative) in the Mozilla Glossary, and its links
+
+### schema
+
+A document, written according to the proscribed structure of the JSON Schema specification, which can be used to validate [instances](#instance).
+
+The rules constituting what schemas are *valid* JSON Schemas, as well as the rules governing their behavior when validating instances, are defined by the JSON Schema specification.
+
+Strictly speaking, according to the specification, schemas are themselves JSON documents, though it is somewhat common for them to be authored or maintained in other languages which are easily translated to JSON, such as YAML.
```

Comment:
_Are_ they JSON docs, or can they merely be _represented_ as JSON docs?

Reply:
In the spec we say the former, from [§4.3](https://json-schema.org/draft/2020-12/json-schema-core.html#section-4.3):

> A JSON Schema document, or simply a schema, is a JSON document used to describe an instance.

(Though I think our "future" should involve loosening that, but that's obviously something to agree on and change)